common/testData/localeIdentifiers/localeCanonicalization.txt

```diff
@@ -50,6 +50,7 @@ adp	;	dz
 afr	;	af
 agp	;	apf
```

Comment:
Note: There's a class called ICUServiceBuilder that plugs CLDR data into ICU formatters, including number and date formatting.